config/i18n-tasks.yml

```diff
@@ -1,4 +1,8 @@
 data:
+  read:
+    - config/locales/%{locale}/*.yml
+  write:
+    - ['{:}.*', 'config/locales/%{locale}/\1.yml']
```

Comment:
This syntax is quite tricky to parse - is it copied and pasted from somewhere? Derived from documentation? Could you explain in the commit message where it came from? Perhaps even an inline comment in the file would be useful too.

Reply:
It is indeed. I've added a commit message explaining where the config was sourced from and what it means.